tests/integration/sync_test.go

```diff
@@ -8787,6 +8787,8 @@ func Test_Sync_KeysAndKeySets(t *testing.T) {
 		name          string
 		kongFile      string
 		expectedState utils.KongRawState
+		update        bool
```

Comment:
I'm wondering if separating the update into a different test would be a good idea, what do you think @Prashansa-K ?

Reply:
Done.